ts/@live-compositor/browser-render/examples/mp4/vite.config.ts

```diff
@@ -0,0 +1,19 @@
+import { defineConfig } from 'vite';
+import { viteStaticCopy } from 'vite-plugin-static-copy';
+
+// https://vitejs.dev/config/
+export default defineConfig({
+  plugins: [
+    viteStaticCopy({
+      targets: [
+        {
+          src: 'node_modules/@live-compositor/browser-render/dist/live-compositor.wasm',
```

Comment:
you can use https://nodejs.org/api/modules.html#modules_require_resolve_request_options to resolve path based on package name